site/content/en/docs/Guides/feature-stages.md

```diff
@@ -27,12 +27,12 @@ The current set of `alpha` and `beta` feature gates:
 | Feature Name                                                                                                          | Gate                     | Default  | Stage   | Since  |
 |-----------------------------------------------------------------------------------------------------------------------|--------------------------|----------|---------|--------|
 | [Custom resync period for FleetAutoscaler](https://github.com/googleforgames/agones/issues/1955)                      | `CustomFasSyncInterval`  | Enabled  | `Beta`  | 1.25.0 |
+| [Graceful Termination for GameServer SDK](https://github.com/googleforgames/agones/pull/2205)                         | `SDKGracefulTermination` | Enabled  | `Beta`  | 1.18.0 |
```

Comment:
Since docs are published immediately, this will need to be wrapped in a feature code:
https://agones.dev/site/docs/contribute/#within-a-page

If you want to move the position in the table, you'll need to wrap the whole table and duplicate the table -- trying to do single rows doesn't work.

Reply:
Do you have any thoughts on the ordering? I am of the opinion that they should be ordered as I think it helps with the toil, but I am not tied to the maturity ordering. We could just do plain alphabetical.

Reply:
No strong opinions here! Happy with whatever you decide.

Reply:
Let's try it like this. It looks a little painful in feature-stages.md but :shrug:, I don't think it's such a common operation that it needs to be optimized and I like the display by maturity instead.